docs/whats_new/v0.4.1.md

```diff
@@ -1,9 +1,192 @@
+
+- Improved memory warning thresholds (#1626) @Innixma
+
+- Added `get_results` and `model_base_kwargs` (#1618) @Innixma
+  - Added `get_results` to searchers, useful for debugging and for future extensions to HPO functionality.
+    Added new way to init a `BaggedEnsembleModel` that avoids having to init the base model prior to initing the bagged ensemble model.
+
+- Update resource logic in models (#1689) @Innixma
+  - Previous implementation would crash if user specified `auto` for resources, fixed in this PR.
+  - Added `get_minimum_resources` to explicitly define minimum resource requirements within a method.
+
+- Updated feature importance default `subsample_size` 1000 -> 5000, `num_shuffle_sets 3` -> 5 (#1708) @Innixma
+  - This will improve the quality of the feature importance values by default, especially the 99% confidence bounds. The change increases the time taken by ~8x, but this is acceptable because of the numerous inference speed optimizations are done since these defaults were first introduced.
+
+- Added notice to ensure serializable custom metrics (#1705) @Innixma
+
+### Bug fixes
+
+- Fixed `evaluate` when `weight_evaluation=True` (#1612) @Innixma
+  - Previously, AutoGluon would crash if the user specified `predictor.evaluate(...)` or `predictor.evaluate_predictions(...)` when `self.weight_evaluation==True`.
+
+- Fixed RuntimeError: dictionary changed size during iteration (#1684, #1685) @leandroimail
+
+- Fixed CatBoost custom metric & F1 support (#1690) @Innixma
+
+- Fixed repeated bagging failing to fit repeats beyond the first when time limit is specified due to `self.model_base` being `None` in the loaded model (#1702) @Innixma
+
+- Fixed HPO not working for bagged models if the bagged model is loaded from disk (#1702) @Innixma
+
+- Fixed Feature importance erroring if `self.model_best` is `None` (can happen if no Weighted Ensemble is fit) (#1702) @Innixma
+
+## Documentation
+
+- updated the text tutorial of cutomizing hyperparameters (#1620) @zhiqiangdon
+  - Added customizeable backbones from the Huggingface model zoo and how to use local backbones.
+
+- Improved implementations and docstrings of `save_pretrained_models` and `convert_checkpoint_name`. (#1656) @zhiqiangdon
+
+- Added cheat sheet to website (#1605) @yinweisu
+
+- Doc fix to use correct predictor when calling leaderboard (#1652) @Innixma
+
+## Miscellaneous changes
+
+- [security] updated `pillow` to `9.0.1`+ (#1615) @gradientsky
+
+- [security] updated `ray` to `1.10.0`+ (#1616) @yinweisu
+
+- Tabular regression tests improvements (#1555) @willsmithorg
+  - Regression testing of model list and scores in tabular on small synthetic datasets (for speed).
+  - Tests about 20 different calls to `TabularPredictor` on both regression and classification tasks, multiple presets etc.
+  - When a test fails it dumps out the config change required to make it pass, for ease of updating.
```

Comment:
We should take a note somewhere to enable this at some point, it currently doesn't run anywhere.